canonical/cloud-init
# VMWARE: search the deployPkg plugin in multiarch dir

## Proposed Commit Message


```
VMWARE: search the deployPkg plugin in multiarch dir

Due to multiarch, the libdeployPkgPlugin.so is deployed into dir
/usr/lib/<multiarch name>/open-vm-tools, we need to add this path
into search_paths.

LP: #1944946
```

## Test Steps
1. Install open-vm-tools in multiarch Ubuntu, verify the customization could executed sucessfully on it.

## Checklist:
 - [ X] My code follows the process laid out in [the documentation](https://cloudinit.readthedocs.io/en/latest/topics/hacking.html)
 - [ X] I have updated or added any unit tests accordingly
